source/solvers/heat_transfer_assemblers.cc

```diff
@@ -486,7 +489,8 @@ HeatTransferAssemblerRobinBC<dim>::assemble_rhs(
                             (h * (T_face - T_inf) +
                              Stefan_Boltzmann_constant * emissivity *
                                (T_face * T_face * T_face * T_face -
-                                T_inf * T_inf * T_inf * T_inf)) *
+                                T_inf * T_inf * T_inf * T_inf) +
+                             heat_flux_bc) *
```

Comment:
```suggestion
                             ) *
```
You added a seperate assembler, so this heat flux should not be in the Robin BC anymore